wpilibj/src/main/java/edu/wpi/first/wpilibj/IterativeRobotBase.java

```diff
@@ -270,14 +270,15 @@ protected void loopFunc() {
     m_watchdog.reset();
 
     // Get current mode
+    var word = new DSControlWord();
```

Comment:
This allocates a new object every loop.  Why not change this to be non-final with an update function?